.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -3,43 +3,29 @@ description: File a bug report for Babelfish Website
 title: "[Bug]: "
 labels: ["bug", "triage"]
 assignees:
-  - amazon-auto
+  - 3manuek
+  - xaviersierra
+  - gopinathpai
+  - nasbyj
 body:
   - type: markdown
     attributes:
       value: |
-        Thanks for taking the time to fill out this bug report!
-  - type: input
-    id: contact
-    attributes:
-      label: Contact Details
-      description: How can we get in touch with you if we need more info?
-      placeholder: ex. email@example.com
-    validations:
-      required: false
+        Thank you for taking the time to fill out this bug report.
   - type: textarea
     id: what-happened
     attributes:
-      label: What happened?
-      description: Also tell us, what did you expect to happen?
-      placeholder: Tell us what you see!
+      label: Describe the bug
+      description: Describe the expected behavior of the website. 
+      placeholder: Tell us what you see.
       value: "A bug happened!"
```

Comment:
Is 'value' displayed onscreen to the user?  If so, how about:

Describe the bug that you've encountered and website behavior.

Reply:
`value` is the example content in the field (the light gray text). Applied the change.